src/Runtime/DrushServiceFinder.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drush\Runtime;
+
+use Drush\Log\Logger;
+use League\Container\Container;
+use Drush\Drush;
+use Symfony\Component\Console\Application;
+use Composer\Autoload\ClassLoader;
+use League\Container\ContainerInterface;
+use Drush\Command\DrushCommandInfoAlterer;
+
+use Psr\Container\ContainerInterface;
+use League\Container\Container as DrushContainer;
+
+/**
+ * Find drush.services.yml files.
+ */
+class DrushServiceFinder
+{
+
```

Comment:
Committed by mistake, but might end up using eventually. Intended as the home for the drush.services.yml discovery methods, but I put that work off and just used the existing code in the Drush Drupal Kernel trait for the initial PoC.

Reply:
Maybe `LegacyServiceFinder` instead?

Reply:
Good suggestion; this code vanishes if support for drush.services.yml is ever removed, and we are going to deprecate that even if it sticks around for b/c.